fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/MetricLoader.java

```diff
@@ -14,7 +14,10 @@
      * Load metrics and populate the metric dictionary.
      *
      * @param metricDictionary  The dictionary that will be loaded with metrics
+     *
+     * @deprecated in favor of loadMetricDictionary(MetricDictionary, DimensionDictionary)
```

Comment:
Indeed it is. Somehow I missed that, or perhaps GitHub was overly aggressive with it's cache ;)